pkg/statsd/parser.go

```diff
@@ -34,10 +38,14 @@ type DatagramParser struct {
 	badLineLimiter *rate.Limiter
 
 	in <-chan []*Datagram // Input chan of datagram batches to parse
+
+	logRawMetric         bool
+	logRawMetricInitOnce sync.Once
+	logRawMetricChann    chan []*gostatsd.Metric
```

Comment:
Nitpick: can we name this `logRawMetricChan` to be consistent with other areas of the codebase and with the `chan` keyword itself?